src/dbcp/metadata/data_mart.py

```diff
@@ -725,3 +725,18 @@
     Column("description", String),
     schema=schema,
 )
+
+county_concrete_mw = Table(
+    "county_concrete_mw",
```

Comment:
Is there a better name for this table? Concrete meaning highly likely mw?

Reply:
that name came from Yujia 🤷🏼I agree though we already have two inscrutable names ("actionable" and "nearly certain") in this realm and adding a third doesn't clarify things to me